planar/planar.go

```diff
@@ -0,0 +1,86 @@
+package planar
+
+import (
+	"math"
+	"math/big"
+)
+
+const Rad = math.Pi / 180
+
+type PointLineDistanceFunc func(line [2][2]float64, point [2]float64) float64
+
+// PerpendicularDistance  provides the distance between a line and a point in Euclidean space.
```

Comment:
Please add a reference to: https://en.wikipedia.org/wiki/Distance_from_a_point_to_a_line

Reply:
"Line defined by two points" section

Reply:
done